vuepress/docs/next/tutorials/samples/customer-portal.md

```diff
@@ -0,0 +1,166 @@
+---
+sidebarDepth: 2
+---
+
+# Entando Customer Portal 
+## Overview
+The Entando Customer Portal enables an organization to quickly provide a modern, self-service customer-facing application for management of subscriptions. It includes a lightweight integration to JIRA Service Management for access to service tickets and a Role Based Access Control (RBAC) design for granting users varying levels of access.
+
+Key Features:
+
+* Customizable service ticket and tracking system with Jira Service Management
+* Role-based access control (RBAC) with Entando Identity Management System
+* Integrated user, customer, project, and subscription management 
+
+This tutorial covers: 
+1. [Installation](#installation)
+2. [Configuration](#configuration)
+4. [Managing the Customer Portal](#managing-the-customer-portal)
+5. [Using the Customer Portal](#using-the-customer-portal)
+6. [Application Details](#application-details)
+7. [Resources](#resources)
+
+## Installation
+### Prerequisites
+* A working instance of Entando running on Kubernetes. See [Getting Started](../../docs/getting-started/) for more information or [install Entando on any Kubernetes provider](../#operations). 
+* Use the Entando CLI command `ent check-env develop` to verify all dependencies
+* Docker installed and running
+
+### Automatic Install with App Builder
+Install the Customer Portal in a few simple steps by integrating the Entando Hub into your App Builder. In the Hub, you can update or roll back versions of a bundle at any time.
+1. From your App Builder, go to the Repository from the left navigation bar. Enter Entando Hub in the Registry field at the top of the page. You will now have access to the Hub.
```

Comment:
If the Hub is already configured, then this is "Select the Entando Hub at the top right of the page." If it's configured they will need to enter both the name and the Hub API URl to set it up initially. That's what we'll want to document in one place.